src/ui/server/controllers/insurance/business/company-details/index.test.ts

```diff
@@ -347,10 +346,24 @@ describe('controllers/insurance/business/companies-details', () => {
           userName: getUserNameFromSession(req.session.user),
           ...pageVariables(mockApplication.referenceNumber, COMPANY_DETAILS_ROUTE),
           SUMMARY_LIST: companyHouseSummaryList(mockCompany),
-          submittedValues,
+          submittedValues: payload,
           validationErrors: {},
         });
       });
+
+      describe('when an extra field is inserted onto the page', () => {
```

Comment:
We don't need to test the extra field/value here, that's the responsibility of `constructPayload`, just need to test that the function is callled

Can be simplified next PR so we unblock things